lib/python/script/utils.py

```diff
@@ -467,3 +471,104 @@ def clock():
     if sys.version_info > (3,2):
         return time.perf_counter()
     return time.clock()
+
+
+def legalize_vector_name(name):
+    if not name:
+        raise ValueError("name cannot be empty")
+    if re.match("[^A-Za-z0-9]", name[0]):
+        name[0] = "_"
+    re.sub("[^A-Za-z0-9_]", "_", name)
```

Comment:
E.g. "snø" would not be "legalized" here, although it is not a valid map name according to v.edit. Only ascii characters are allowed.

Reply:
`A-Za-z` really matches just ASCII as far as I can tell, so that should be fine. There was however, much bigger problem, missing `name = `.